docs/reference/config-params.md

```diff
@@ -280,462 +324,543 @@ Otherwise it will default to `1`.
 
-> kev.workload.max-memory:
+> x-k8s.workload.resource.maxMemory:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.max-memory: 0.3Gi
+    x-k8s:
+      workload:
+        resource:
+          maxMemory: 0.3Gi
 ...
 ```
 
-## kev.workload.liveness-probe-type
+## x-k8s.workload.livenessProbe
+
+Defines the workload's liveness probe.
 
-Defines the workload's liveness probe type. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer from the information specified in the compose file.
+### x-k8s.workload.livenessProbe.type
 
-Kev uses the following heuristics to derieve that information for each service:
+This setting defines the workload's liveness probe type. Kev will attempt to infer from the information specified in the compose file.
+
+Kev uses the following heuristics to derive that information for each service:
 
 If compose file(s) specifies the `healthcheck.disable` attribute key in a service config it will set the probe type to `none`.
 Otherwise it'll default to `exec` (liveness probe active!)
 
-### Default: `exec`
+#### Default: `exec`
 
-### Possible options: none, exec, http, tcp.
+#### Possible options: none, exec, http, tcp.
 
-> kev.workload.liveness-probe-type:
+> x-k8s.workload.livenessProbe.type:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-type: none
+    x-k8s:
+      workload:
+        livenessProbe:
+          type: none
 ...
 ```
 
-## kev.workload.liveness-probe-command
+### x-k8s.workload.livenessProbe.exec.command
 
-Defines the liveness probe command to be run for the workload. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer the command from the information specified in the compose file.
+Defines the liveness probe command to be run for the workload when the type is `exec`.
+See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer the command from the information specified in the compose file.
 
-Kev uses the following heuristics to derieve that information for each service:
+Kev uses the following heuristics to derive that information for each service:
 
 If compose file(s) specifies the `healthcheck.test` attribute key in a service config it will use its value.
 If probe is not defined it will prompt the user to define one by injecting generic echo command.
 
-### Default: echo "prompt user to define the probe"
+#### Default: echo "prompt user to define the probe"
```

Comment:
Is it still a default?

Reply:
Updated to: `echo "<generic prompt text>" ` as we don't care what the actual text is.